### DISCUSSION TITLE
How to keep filtering mentions with spaces?

### DISCUSSION THREAD

**Discussion by rfgamaral** [ueberdosis/tiptap#2032] [General Questions] [answered]
The current `@tiptap/extension-mention` doesn't allow one to filter mentions with spaces. For instance, say I have a list of names and I start to type `@keanu r`, as soon as I press `<space>` after `@keanu`, the mentions dropdown disappears, but I was hoping to match the entry "Keanu Reeves".

Can this be fixed by changing the code in `@tiptap/extension-mention`, or does it need deeper changes to the Suggestion utility?

**Comment by hanspagel** [comment, marked as answer]
Pass `allowSpaces: true` to the suggestion utility!

https://tiptap.dev/api/utilities/suggestion#allow-spaces

**Reply by rfgamaral** [reply]
I don't know how I missed that, I feel dumb. Sorry for wasting your time with such a basic thing...
